draft-fett-oauth-sd-jwt-vc-types.md

```diff
@@ -402,25 +406,91 @@ MUST contain at least one of the following properties:
 
 # Claim Metadata {#ClaimMetadata}
 
-The `claims` property is an object containing information about particular claims
-for displaying and validating the claims.
+The `claims` property is an array of objects containing information about
+particular claims for displaying and validating the claims.
 
-The object MAY contain a property for each claim that is supported by the type.
-The property name MUST be a JSON Pointer [@!RFC6901] pointing to the claim in the
-credential. The property value MUST be an object containing the following
-properties:
+The object MAY contain an object for each claim that is supported by the type.
+The property value MUST be an object containing the following properties:
```

Comment:
I've read this bit several times and it just doesn't seem right (i.e. "object MAY contain an object" wouldn't be valid JSON and I don't think is what you are trying to say anyway and there's no property value to be an object at this level). Here is an attempt to adjust the words to say what I think you were trying to say:

```suggestion
The array MAY contain an object for each claim that is supported by the type.
Each object contains the following properties:
```